stable/gangway/templates/configmap.yaml

```diff
@@ -10,5 +10,11 @@ metadata:
 data:
   gangway.yaml: |
     {{- .Values.gangway | toYaml | nindent 4 }}
+    {{- if .Values.trustedCACert }}
+    trustedCAPath: /gangway/rootca.crt
+  rootca.crt: |
+    {{ .Values.trustedCACert | nindent 4}}
```

Comment:
I would probably use `{{-` here instead of `{{`. The as it is now is:

```
  rootca.crt: |

    -----BEGIN CERTIFICATE-----
    SOMECERTDATA
    -----END CERTIFICATE-----
```

Which might work, but looks a little strange, by adding the `-` you get the slightly nicer:

```
  rootca.crt: |
    -----BEGIN CERTIFICATE-----
    SOMECERTDATA
    -----END CERTIFICATE-----
```